master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -41,11 +40,8 @@ This document describes three installation options for Calico using Kubernetes A
 
 Ensure you have a cluster which meets the above requirements.  There may be additional requirements based on the installation option you choose.
 
-> **NOTE**
->
-> There is currently no upgrade path to switch between different installation options.  Therefore,
-> if you are upgrading from Calico v2.1, use the [Calico policy-only with user-supplied networking](#2-calico-policy-only-with-user-supplied-networking) installation instructions
-> to upgrade Calico policy-only which leaves the networking solution unchanged.
+<div class="alert alert-info" role="alert"><b>Note</b>: There is currently no upgrade path to switch between different installation options.  Therefore, if you are upgrading from Calico v2.1, use the <a href="#2-calico-policy-only-with-user-supplied-networking">Calico policy-only with user-supplied networking</a> installation instructions to upgrade Calico policy-only which leaves the networking solution unchanged.</div>
```

Comment:
I think at this point, Calico v2.1 is irrelevant here.  There's no upgrade path between the two versions period - no matter what version of Calico is in use.

We mention v2.1 here because the version the second manifest was introduced in was v2.2

Reply:
@caseydavenport Are you suggesting that we might be able to delete this note?

Reply:
Eh, I changed my mind. Let's leave it for now.  :)